cryptol-remote-api/python/requirements.txt

```diff
@@ -1,30 +1,81 @@
-alabaster==0.7.12
-argo-client==0.0.4
-Babel==2.8.0
-BitVector==3.4.9
-certifi==2020.6.20
-chardet==3.0.4
-docutils==0.16
-idna==2.10
-imagesize==1.2.0
-Jinja2==2.11.3
-MarkupSafe==1.1.1
-mypy==0.812
-mypy-extensions==0.4.3
-packaging==20.4
-Pygments==2.7.1
-pyparsing==2.4.7
-pytz==2020.1
-requests==2.24.0
-six==1.15.0
-snowballstemmer==2.0.0
-Sphinx==3.2.1
-sphinxcontrib-applehelp==1.0.2
-sphinxcontrib-devhelp==1.0.2
-sphinxcontrib-htmlhelp==1.0.3
-sphinxcontrib-jsmath==1.0.1
-sphinxcontrib-qthelp==1.0.3
-sphinxcontrib-serializinghtml==1.1.4
-typed-ast==1.4.2
-typing-extensions==3.7.4.3
-urllib3==1.25.10
+argo-client==0.0.4; python_version >= "3.7" \
```

Comment:
How did you generate this `requirements.txt` file, out of curiosity? This looks way too intricate to have been written by hand.

Reply:
`poetry export -f requirements.txt > requirements.txt`

Reply:
Wow, you taught me something about `poetry`!

Per the discussion in GaloisInc/saw-script#1152, `pyproject.toml` might be somewhat underspecified at the moment since it doesn't mention Sphinx in its `dev-dependencies`, so we might be losing some functionality from `requirements.txt` by generating it with Poetry. Perhaps we should investigate generating the Sphinx documentation with `poetry` as well (but perhaps not in this PR to avoid scope creep).

Reply:
This has been spun off into https://github.com/GaloisInc/cryptol/issues/1138.